docs/document-level-security/SALESFORCE.md

```diff
@@ -0,0 +1,57 @@
+### Setting up the Salesforce connector
+
+See the [Developer guide](../../docs/DEVELOPING.md) for setting up connectors.
+
+## Document level security
+
+Document level security (DLS) enables you to restrict access to documents based on a user'­s permissions. This feature is available by default for the Salesforce connector.
+Salesforce connector DLS supports for both standard & custom objects.
+
+Refer to [document level security](https://www.elastic.co/guide/en/enterprise-search/master/dls.html) for more information.
+
+Salesforce allows users to set permissions in different ways i.e. via Profiles, Permission sets and Permission set Groups. 
+
+Refer this tutorial to get more idea on setting the permissions - [link](https://howtovideos.hubs.vidyard.com/watch/B1bQnMFg2VyZq7V6zXQjPg#:~:text=This%20is%20a%20must%20watch,records%20in%20your%20Salesforce%20organization.)
+
+### Set Permissions using Profiles
+
+Follow below steps to set permissions via Profiles:
+1. From the setup page, go to `Administration` section => `Users` => `Profiles` and create a new profile
+2. Choose `Read Only` or `Standard User` for the Existing Profile dropdown, give a name to the profile and save it. By default, `Read Only` or `Standard User` have the read permission to access all standard objects. There can be some more profiles which do have a read access to standard objects but these are some of them.
+3. Now, edit the newly created profile and under `Object Permissions`, assign at least a `Read` access to the standard objects and custom objects you want to ingest into ElasticSearch.
+
+**Note:** If users specify advanced sync rules then they need to assign a `Read` access for that specific object in the profile.
```

Comment:
> need to assign a `Read` access for that specific object

Is this true for both standard and custom objects?

Reply:
Yes, this is applicable to both standard and custom objects